concordium-sdk/src/main/java/com/concordium/sdk/transactions/Payload.java

```diff
@@ -91,7 +91,9 @@ private static UInt64 calculateEnergyCost(int noOfSignatures,
     public enum PayloadType {
         TRANSFER,
         TRANSFER_WITH_MEMO,
-        REGISTER_DATA
+        REGISTER_DATA,
+        UPDATE_SMART_CONTRACT_INSTANCE,
```

Comment:
Please just name it `Update` so it aligns with `concordium-base`. I think that will make it easier for future maintainers.

Reply:
Done